sound/soc/sof/sof-audio.h

```diff
@@ -106,7 +106,7 @@ struct snd_sof_route {
 };
 
 /* ASoC DAI device */
-struct snd_sof_dai {
+struct snd_sof_dai_link {
 	struct snd_soc_component *scomp;
 	const char *name;
 	const char *cpu_dai_name;
```

Comment:
If this struct is actually describing a link with potentially multiple DAIs on it, what do the `.cpu_dai_name`, `.comp_dai` and `.dai_config` now refer to?